ra/registration-authority.go

```diff
@@ -225,6 +228,6 @@ func (ra *RegistrationAuthorityImpl) OnValidationUpdate(authz core.Authorization
 		authz.Expires = time.Now().Add(365 * 24 * time.Hour)
 	}
 
-	// Finalize the authorization
-	ra.SA.FinalizeAuthorization(authz)
+	// Finalize the authorization (error ignored)
+	_ = ra.SA.FinalizeAuthorization(authz)
```

Comment:
Should I add a TODO to all the ignored errors I found via errcheck? I would've done it if it was my project, but I didn't want to clutter your code with TODOs :)